.env.example

```diff
@@ -12,6 +12,32 @@ ELASTICSEARCH_PORT=9200
 ELASTICSEARCH_URL=http://elasticsearch:9200
 MONGOHQ_URL=mongodb://localhost:27017/positron
 
+##
+# Set the DATADOG_AGENT_HOSTNAME to some value and start a local agent via Docker
+# with a `--name` equal to DATADOG_AGENT_HOSTNAME to perform local development
+# against Positron's configuration with Data Dog.
+#
+# IMPORTANT: Set the `-e DD_HOSTNAME` when running the agent to something that'll
+# uniquely identity your computer. For example, `daniel-phoenix`. As Data Dog
+# is a shared service, this will allow you to easily isolate your work in the
+# Data Dog UI.
+#
+# A docker invocation that worked for running DD agent on my mac:
+#
+#```
+# docker run --name datadog-agent \
+#  --network hokusai_default \
+#  -v /var/run/docker.sock:/var/run/docker.sock:ro
+#  -v /proc/:/host/proc/:ro \
+#  -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+#  -e DD_API_KEY=[GET-FROM-DD-UI] \
+#  -e DD_APM_ENABLED=true
+#  -e DD_HOSTNAME=daniel-phoenix
```

Comment:
`- e DD_HOSTNAME=$(whoami)-phoenix` ?

Reply:
Yeah, or even a `"$(whoami)-$(hostname -s)"` - i can update.

Reply:
Maybe the hostname is a bit of overkill but /shrug, it was helpful for me to ensure that metrics were emanating from my computer